File: src/Example3Java/EX22.java
package Example3Java;

import java.util.Scanner;

public class EX22 {
    public static void main(String[] args) {
        System.out.println("Enter row : ");
        int row = new Scanner(System.in).nextInt();
        System.out.println("Enter colum : ");
        int colum = new Scanner(System.in).nextInt();

        int[][] x = new int[row][colum];
        for (int i = 0; i < row; i++) {
            for (int j = 0; j < colum; j++) {
                System.out.println("Enter Array : ");
                x[i][j] = new Scanner(System.in).nextInt();
            }
        }
    }
}

File: src/Example3Kotlin/EX22.kt
import java.util.*

object EX22 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter row : ")
        val row = Scanner(System.`in`).nextInt()
        print("Enter colum : ")
        val comlum = Scanner(System.`in`).nextInt()
        val x = Array(row) { IntArray(comlum) }

        for (i in 0 until  row) {
            for (j in 0 until comlum) {
                print("Enter Array : ")
                x[i][j] = Scanner(System.`in`).nextInt()
            }
        }
        println(x.contentDeepToString())
    }
}
